charts/drax/values.yaml

```diff
@@ -517,7 +517,20 @@ grafana:
           isDefault: false
           name: Loki
           type: loki
-          url: http://{{ .Release.Name }}-loki.{{ .Release.Namespace }}:3100
+          url: http://{{ .Release.Name }}-loki.{{ .Release.Namespace }}:3101
+          basicAuth: true
+          basicAuthUser: logs
+          secureJsonData:
+            basicAuthPassword: accelleran
+        - access: proxy
+          isDefault: false
+          name: Loki-notifications
+          type: loki
+          url: http://{{ .Release.Name }}-loki.{{ .Release.Namespace }}:3101
+          basicAuth: true
+          basicAuthUser: notifications
+          secureJsonData:
+            basicAuthPassword: accelleran
```

Comment:
Could we inject this with the secret we create above? I'm mainly thinking that we should be looking at a single source for each password within our helm chart. Otherwise there would be multiple sources for the password which need to be kept in sync...

Reply:
For grafana specifically we maybe would need to be looking at the operator...